docs/sources/logql/metric_queries.md

```diff
@@ -114,6 +115,58 @@ sum by (org_id) (
 
 This calculates the amount of bytes processed per organization ID.
 
+```logql
+buckets_over_time((.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10),
```

Comment:
This is great but I think it would be better if we could find bucket automatically like sparse histogram https://grafana.com/blog/2021/11/03/how-sparse-histograms-can-improve-efficiency-precision-and-mergeability-in-prometheus-tsdb/

Reply:
I think if finding buckets is doable as per the doc I shared then I would love to have only this